bench/Evaluation.hs

```diff
@@ -34,47 +34,38 @@ import           System.Path ((</>))
 
 callGraphProject' :: ( Language.SLanguage lang
                      , HasPrelude lang
-                     , AccessControls (term Loc)
-                     , Declarations (term Loc)
-                     , Evaluatable (Base (term Loc))
-                     , FreeVariables (term Loc)
-                     , HasSpan (term Loc)
-                     , Ord (term Loc)
-                     , Recursive (term Loc)
-                     , Show (term Loc)
-                     , VertexDeclaration term
```

Comment:
The really nice thing about abstract parsing is that you get to eliminate a huge number of constraints because they’re implied by the map of parsers you’re selecting from.